log/src/main/java/net/md_5/bungee/log/BungeeLogger.java

```diff
@@ -26,6 +26,7 @@ public BungeeLogger(String loggerName, String filePattern, ConsoleReader reader)
         try
         {
             FileHandler fileHandler = new FileHandler( filePattern, 1 << 24, 8, true );
+            fileHandler.setLevel( Level.parse( System.getProperty( "net.md_5.bungee.file-log-level", "FINEST" ) ) );
```

Comment:
I'd suggest to use `INFO` as default file logging level as well. The standard user doesn't need FINE and FINEST log messages.
```suggestion
            fileHandler.setLevel( Level.parse( System.getProperty( "net.md_5.bungee.file-log-level", "INFO" ) ) );
```